easybuild/easyconfigs/s/Single-cell-python-bundle/Single-cell-python-bundle-2024.02-foss-2023a.eb

```diff
@@ -0,0 +1,181 @@
+easyblock = 'PythonBundle'
+
+name = 'Single-cell-python-bundle'
+version = '2024.02'
+
+homepage = (None)
+description = """Bundle of tools for single-cell sequence analysis."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+# builddependencies = [('poetry', '1.5.1')]
```

Comment:
@pavelToman `poetry` is required for installing `pypi-latest`, so you'll have to uncomment this?

See failed test report in https://gist.github.com/boegelbot/caeab59f44f6611f00bff1c7073031c9